test/e2e/apimachinery/aggregator.go

```diff
@@ -74,7 +74,17 @@ var _ = SIGDescribe("Aggregator", func() {
 	ginkgo.BeforeEach(func() {
 		c = f.ClientSet
 		ns = f.Namespace.Name
-		aggrclient = f.AggregatorClient
+
+		if aggrclient == nil {
```

Comment:
@timothysc had to do this.
otherwise having the LoadConfig()/NewForConfig(config) outside of BeforeEach panics.